src/psyclone/tests/dynamo0p3_test.py

```diff
@@ -51,7 +51,7 @@
 from psyclone.psyGen import PSyFactory, GenerationError
 from psyclone.dynamo0p3 import DynKernMetadata, DynKern, DynLoop, \
     FunctionSpace, VALID_STENCIL_TYPES, DynHaloExchange, \
```

Comment:
Not sure if this is down to this issue but pylint complains that DynHaloExchange doesn't need to be imported.